sdcm/cluster.py

```diff
@@ -5469,12 +5469,22 @@ def install_scylla_monitoring_prereqs(self, node):  # pylint: disable=invalid-na
 
         if node.distro.is_rhel_like:
             node.install_epel()
-            node.install_package(package_name="unzip wget python36 python36-pip")
-            prereqs_script = dedent("""
+            node.install_package(package_name="unzip wget python3 python3-pip")
+            # get-docker.sh doesn't support Rocky, thus docker should be installed from repo via dnf install
+            if node.distro.is_rocky:
+                install_docker_cmd = dedent("""
+                    dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
```

Comment:
nitpick: isn't those command also works on all of the rest rhel_like ?

Reply:
I believe it should but I haven't checked it and decided to go with the approach above to minimize possible regression.